lib/basedriver/commands/session.js

```diff
@@ -4,6 +4,7 @@ import log from '../logger';
 import { errors } from '../../protocol';
 import UUID from 'uuid-js';
 import { processCapabilities } from '../capabilities';
+import prettyJson from 'prettyjson';
```

Comment:
what is the point in using third-party dependency? we can still add indentation parameters to JSON.stringify call